cmd/generate/params_test.go

```diff
@@ -0,0 +1,173 @@
+package generate_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/blang/semver/v4"
+	"github.com/wakatime/semver-action/cmd/generate"
+
+	"github.com/alecthomas/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadParams_Prefix(t *testing.T) {
+	os.Setenv("INPUT_PREFIX", "v")
+	os.Setenv("INPUT_DRY_RUN", "true")
+
```

Comment:
As you intend to cleanup the env variables at the end, the standard go way would be to use `defer` here. The the creation and removal of the env variable is next to each other.
```suggestion
	os.Setenv("INPUT_PREFIX", "v")
	defer os.Unsetenv("INPUT_PREFIX")

	os.Setenv("INPUT_DRY_RUN", "true")
	defer os.Unsetenv("INPUT_DRY_RUN")

```

Reply:
Or in a function:
```suggestion
	os.Setenv("INPUT_PREFIX", "v")
	os.Setenv("INPUT_DRY_RUN", "true")
	defer func() {
	    os.Unsetenv("INPUT_PREFIX")
          os.Unsetenv("INPUT_DRY_RUN")
      }()

```